nodebuilder/share/module.go

```diff
@@ -29,7 +30,12 @@ func ConstructModule(tp node.Type, cfg *Config, options ...fx.Option) fx.Option
 		fx.Options(options...),
 		fx.Provide(discovery(*cfg)),
 		fx.Provide(newModule),
-		fxutil.ProvideAs(getters.NewIPLDGetter, new(share.Getter)),
+		// TODO: Configure for light nodes
+		fx.Provide(
```

Comment:
pls only add to light nodes, bridges and fulls don't need to run nd client

Reply:
they do, and it is required for shrexgetter. what if bridges/fulls havent synced the data yet